### ISSUE TITLE
Please explain plugin usefulness

### ISSUE THREAD

**Issue by pcav** [skeenp/QGIS3-getWKT#1]
Simply copying a feature copies also its WKT representation.
What does the plugin add to it?
Thanks.

**Comment by skeenp** [comment]
Copying a feature copies the wkt and all attributes of a feature. To extract the wkt from that is a difficult job when there are more than a couple of fields. A dialog which isolates just the WKT of an feature makes it easy for us in update statements in a database using st_geomfromtext (or equilivant). This allows the easy update of geometry from a feature in another layer.

If more of a point of difference is needed, I will implement to planned features noted in the readme and resubmit. That being supporting EWKT for PostGis and providing the ability to surround the wkt with the geomfromtext call ready for insertion into a sql statement.

There is obviously a call for the functionality as the QGIS 2 version attracted around 14000 downloads https://plugins.qgis.org/plugins/getWKT/

I have contacted the developer of the original plugin who does not intend on updating his version to QGIS3. The only thing i was unsure of was if this should be added to that plugin as a version, or as its own QGIS3 version. I chose the latter but am open to suggestions.
